common/src/main/java/io/netty/util/concurrent/AbstractScheduledEventExecutor.java

```diff
@@ -244,21 +244,20 @@ protected void validateScheduled(long amount, TimeUnit unit) {
         // NOOP
     }
 
+    final void scheduleFromEventLoop(final ScheduledFutureTask<?> task) {
+        scheduledTaskQueue().add(task.setId(++nextTaskId));
```

Comment:
Should we ensure we will increment another time when `nextTaskId` == 0 after incrementing it (which could happen in theory) as we tread `this.id == 0` special below ?

Reply:
@normanmaurer this is why I used `++n` rather than `n++`, so the first value actually used should be 1 rather than 0, right?

Reply:
sure but it may overflow and so reach 0 again ;)

Reply:
@normanmaurer but it's a `long` :)

Reply:
That’s why I said in theory ;)

Reply:
Even if you were to schedule a new future task every single nanosecond, it would still require a JVM to be running for almost 600 years without restarting for this to be a problem...

Would you be ok with adding a "this won't overflow" comment instead? :)